hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/request/MetricAggregationRequestBuilder.java

```diff
@@ -22,4 +22,10 @@ MetricAggregationRequest build(
       AttributeModel attribute,
       AttributeModelMetricAggregationType aggregationType,
       List<Object> arguments);
+
+  MetricAggregationRequest build(
+          AttributeModel attribute,
+          AttributeModelMetricAggregationType aggregationType,
+          List<Object> arguments,
+          List<SelectedField> selections);
```

Comment:
Which selections are these meant to be? the sub-selections of the aggregation? If they're needed, to resolve baselines (which likely they are), how do we make the other APIs compatible without silently breaking. In other words, doesn't this belong as a modification of the existing API?

Reply:
This is the problem that I'm trying to point out though - existing calls won't break according to the compiler, but if they contain baselines those baselines won't be fetched so it will functionally be broken. We should modify the existing api and force ourselves to explicitly make the decision about baselines on the caller side.